src/libstd/path/posix.rs

```diff
@@ -59,6 +60,12 @@ pub fn is_sep(c: char) -> bool {
     c == SEP
 }
 
+impl fmt::Show for Path {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "Path {{ {} }}", self.display())
```

Comment:
Why put a `Path` out in front? I would expect this to basically do what `display` does today.